core/capabilities/launcher.go

```diff
@@ -387,7 +387,8 @@ func (w *launcher) addToRegistryAndSetDispatcher(ctx context.Context, capability
 }
 
 var (
-	defaultTargetRequestTimeout = time.Minute
+	// TODO: make this configurable
+	defaultTargetRequestTimeout = 8 * time.Minute
```

Comment:
Is this the timeout value that gets mapped to `capability.Execute` fn ctx?

Please notice that the Aptos WT retries RPC calls with exponential backoff up until timeout (`capability.Execute` fn ctx timeout): https://github.com/smartcontractkit/chainlink-internal-integrations/blob/develop/aptos/relayer/write_target/write_target.go#L265

I've pinged you and @archseer in a PR here to try to confirm these timeouts are set and not unbounded: https://github.com/smartcontractkit/chainlink-internal-integrations/pull/276/files#r1871803009

-- 

Now, 8 min, what's the use case to try retrying for so long?

Reply:
But yeah we need to have a separate discussion about different re-try mechanisms and how to make sure they work together well...